### DISCUSSION TITLE
It happened again (Qtile doesn't start and can't load the default config)

### DISCUSSION THREAD

**Discussion by greg086** [qtile/qtile#5123] [General]
Hi, I posted here a few weeks ago about qtile refusing to load [https://github.com/qtile/qtile/discussions/5099](url) (original thread is locked). The problem seemed to be with my home directory and was solved by reinstalling linux with a fresh home. Now I have the same issue with the new install (getting kicked back to sddm).

Last time I was asked for my .xsession-errors file but couldn't provide it because my distro, Arco was not logging there. Now I'm on EndeavourOS and it's logging there so I'm attaching the file.

[xsession-errors.txt](https://github.com/user-attachments/files/18158164/xsession-errors.txt)
There are attribute errors mentioning qtile at the end of the file.

Thanks for the help in advance.

**Comment by greg086** [comment]
I just realized the xsession-errors file is useless because it was last written to weeks ago. Guess I have to reinstall again.

**Reply by elParaguayo** [reply]
Is there anything in qtile's own log?

**Reply by greg086** [reply]
No, not on the day it happened.

**Comment by TheHighGroundWins** [comment]
I have had similar issue were my login manager gdm would not. And I wasn't aware that it was because of my faulty config file, since it would still boot on my config but with StatusNotifier not working.

It showed up in my qtile logs, but instead of showing errors in my config it kept showing errors within qtile's backend. Possibly it still ran because the file was somewhat readable.
